src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+
+export default class Drawer extends Component {
+    constructor(props) {
+        super(props)
+        
+        this.state={
+            name:''
+        }
+
+        this.handleSubmit=this.handleSubmit.bind(this)
+        this.formEdit=this.formEdit.bind(this)
+        this.clear=this.clear.bind(this)
+    }

```

Comment:
Binding wouldn't be an issue with prop drilling to pull down the functions from the App component where they modify the state.